PhysicsTools/HeppyCore/scripts/heppy_batch.py

```diff
@@ -211,7 +211,7 @@ def batchScriptIC(jobDir):
 python {cmssw}/src/PhysicsTools/HeppyCore/python/framework/looper.py pycfg.py config.pck
 echo
 echo 'sending the job directory back'
-mv Loop/* ./
+cp -r Loop/* ./ && rm -r Loop
```

Comment:
why this change?
usually move is cheaper than copy + delete (and doesn't use up twice the space on disk or afs while it is performed)